package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "capital-framework",
-  "version": "4.7.0",
+  "version": "4.7.2",
```

Comment:
Ah good call. Since *only* CF was updated (no components) I manually bumped it. But then I also included `[patch]` in the changelog. So I should have just done one or the other.